app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/WalletGift.kt

```diff
@@ -0,0 +1,116 @@
+package com.aptoide.android.aptoidegames.drawables.icons
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.size
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.PathFillType.Companion.NonZero
+import androidx.compose.ui.graphics.SolidColor
+import androidx.compose.ui.graphics.StrokeCap.Companion.Butt
+import androidx.compose.ui.graphics.StrokeJoin.Companion.Miter
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.graphics.vector.ImageVector.Builder
+import androidx.compose.ui.graphics.vector.group
+import androidx.compose.ui.graphics.vector.path
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+
+@Preview
+@Composable
+private fun TestWalletGift() {
+  Image(
+    imageVector = getWalletGift(),
+    contentDescription = null,
+    modifier = Modifier.size(240.dp)
+  )
+}
+
+fun getWalletGift(): ImageVector = Builder(
+  name = "FeaturedSeasonalAndGifts",
+  defaultWidth = 17.0.dp,
+  defaultHeight = 16.0.dp,
+  viewportWidth = 17.0f,
+  viewportHeight = 16.0f
+).apply {
+  group {
+    path(
+      fill = SolidColor(Color(0xFF1E1E26)),
```

Comment:
Let's not hardcode here but provide it through arguments.
Better set one of the new colours in the place it is used.

Reply:
fixed